### ISSUE TITLE
Util Types

### ISSUE THREAD

**Issue by akatz** [embroider-build/embroider#616]
I am trying to convert an addon written in typescript and getting a warning on import about 
```
import { ensureSafeComponent } from "@embroider/util";
```
The error is
```
Could not find a declaration file for module '@embroider/util'. '/Users/akatz/Source/ember-basic-dropdown/node_modules/@embroider/util/index.js' implicitly has an 'any' type.
  Try `npm install @types/embroider__util` if it exists or add a new declaration (.d.ts) file containing `declare module '@embroider/util';`ts(7016)
````
I was surprised to find out that this package has no types. 

I can happily just type it in my project but I was wondering about the larger decision and if I should try and write them to upstream them to embroider in the future.

**Comment by simonihmig** [comment]
> I can happily just type it in my project but I was wondering about the larger decision and if I should try and write them to upstream them to embroider in the future.

Yes, a PR would be highly welcome! The Ember addons included in this monorepo aren't written in TypeScript, so we need to add declaration files manually (see https://github.com/embroider-build/embroider/pull/598)

**Comment by simonihmig** [comment]
Closed by #633
